src/connect-zwa-2/getting-started/override-radio-frequency-region-and-transmit-power.md

```diff
@@ -17,7 +17,7 @@ If you use the **Z-Wave JS** add-on, you do not need to do anything.
 - The Z-Wave JS add-on automatically adjusts the region based on the location defined for your Home Assistant hub.
   - This means radio frequency and transmit power level are set automatically for your region.
   - If you don't know whether you use the **Z-Wave JS** add-on:
-    - Go to **Settings** > **Add-ons** and look for the **Z-Wave JS** add-on.
+    - Go to **Settings** > **Add-ons**](https://my.home-assistant.io/redirect/supervisor/) and look for the [**Z-Wave JS**](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_zwave_js) add-on.
```

Comment:
```suggestion
    - Go to **Settings** > **Add-ons**](https://my.home-assistant.io/redirect/supervisor/) and look for the [**Z-Wave JS**](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_zwave_js) add-on.
    - In the top right corner, it will be a green "play" icon if you have it installed and running.
```